.github/workflows/beta.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
Maybe because the files are mixing hashes and versions? Let me send an update beta.yml to use the v2 hash and test the theory.